packages/cliqz-component-styles/stories/index.story.tsx

```diff
@@ -0,0 +1,36 @@
+import { storiesOf } from '@storybook/react';
+import React from 'react';
+
+const mergeExample = `
+  const base = {
+    container: {
+      margin: 2,
+    },
+    text: {
+      color: 'black',
+    },
+  };
+
+  const customization = {
+    container: {
+      margin: 5,
+    },
+  };
+
+  merge(base, customization);
+  // {
```

Comment:
maybe we can get rid of this

Reply:
This is being shown on a Storybook as an example of usage.

Reply:
Ahh, cool. :)